lib/metasploit/framework/mssql/client.rb

```diff
@@ -357,12 +406,15 @@ def mssql_login(user='sa', pass='', db='', domain_name='')
           info = {:errors => []}
           info = mssql_parse_reply(resp, info)
 
-          disconnect
 
           return false if not info
           info[:login_ack] ? true : false
         end
 
+        #do we still need this?
```

Comment:
Happy to remove in the second PR; but pretty sure we should be good to nuke this